doc/nrf/known_issues.rst

```diff
@@ -2750,17 +2762,17 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0 v2-3-0
```

Comment:
This line does not need the change, since DRGN-18424 has been fixed already in NCS 2.4.